images/chaos-kernel/Dockerfile

```diff
@@ -36,6 +36,6 @@ WORKDIR /
 
 RUN git clone https://github.com/chaos-mesh/bpfki
 WORKDIR /bpfki/build
-RUN cmake .. && make -j && mv bin/bpfki /usr/local/bin/ && mv examples/fail* /usr/local/bin/
+RUN cmake .. && make -j 16 && mv bin/bpfki /usr/local/bin/ && mv examples/fail* /usr/local/bin/
```

Comment:
Maybe 4 is enough, for github action (as it has only 2 cpus).

Reply:
how about `make -j $(nproc)`

Reply:
or `$[$(nproc) + 1]` or `$[$(nproc) * 2]`?

Reply:
I noticed here is an arg called `MAKE_JOBS`, I would use this arg with default value `4`.